### ISSUE TITLE
"Swipe up" gesture in fullscreen

### ISSUE THREAD

**Issue by HTSamurai** [TeamNewPipe/NewPipe#5950]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

What I want is the swipe up gesture from main-app YouTube that pulls up a menu. This could take form of a new button or gesture because swiping is taken by volume/brightness, and sometimes on some phones home/google assistant.

#### Is your feature request related to a problem? Please describe it

The menu that comes up on main YouTube was nice to have and while I've never used it myself, I have on other phones. It would be great to have this.

#### Additional context
Time for our regularly scheduled *Mock-ups*
![IMG_20210329_083742](https://user-images.githubusercontent.com/71007160/112803394-21543800-906b-11eb-8e7b-a52ed7df0ac4.png)
![IMG_20210329_084505](https://user-images.githubusercontent.com/71007160/112803421-2913dc80-906b-11eb-8e50-524775173cda.jpg)

#### How will you/everyone benefit from this feature?

If people "migrate" to newpipe and find this missing it *might* just put them off. Who knows.

**Comment by SameenAhnaf** [comment]
Swipe up from bottom edge could be added for "Related videos". If swiping up did not occur from any edge, it could be regarded for volume or brightness.

**Comment by SameenAhnaf** [comment]
Swipe up from bottom edge could be added for "Related videos". The same way how it happens for notification panel if swiped down from upper edge. If swiping up did not occur from any edge, it could be regarded for volume or brightness. No need to add extra buttons on video interface.

**Comment by HTSamurai** [comment]
> Swipe up from bottom edge could be added for "Related videos". The same way how it happens for notification panel if swiped down from upper edge. If swiping up did not occur from any edge, it could be regarded for volume or brightness. No need to add extra buttons on video interface.

On some phones swiping from edges acts as a back button, a home button or google assistant if they use fullscreen gestures, like my own

**Comment by 0Karakurt0** [comment]
> fullscreen gestures

Despite that official YouTube App made it work, and made it work nicely.
I'm not sure hot to integrate brightness/volume gestures tho ('cuz I hate them)
Maybe if you're in full screen then you control brightness/volume, and when you see UI swipe up pulls out recommendations

**Comment by SameenAhnaf** [comment]
Won't you be suited by https://github.com/TeamNewPipe/NewPipe/issues/5967 ? It is expected that one would like to browse related videos only after finishing the video.

**Comment by HTSamurai** [comment]
If YouTube made it work, newpipe can do it better. (Take that google)

**Comment by SameenAhnaf** [comment]
What do you think about "Swiping up the bottom interface"? As you can see in the video, swiping down the upper interface switches full screen to mini player. This gesture should not work if interface is not shown as that may cause conflict with brightness and volume gesture.

The space of bottom interface is big enough and quite  dim enough. So, accidental touch should not occur in seek bar.

https://user-images.githubusercontent.com/79245641/116769323-02c3d100-aa5d-11eb-8661-2499c3169a85.mp4

**Comment by opusforlife2** [comment]
> The menu that comes up on main YouTube was nice to have and while I've never used it myself, I have on other phones. It would be great to have this.

What do you want from the menu, specifically? Related videos? The ability to subscribe? Sharing?

**Comment by 0Karakurt0** [comment]
It would be nice to have related videos there, and so that they responded to longpress in the same way.
Essentially I would love not to have to rotate my phone to choose the next video in a queue from related ones.
